### DISCUSSION TITLE
Includes / Excludes Help

### DISCUSSION THREAD

**Discussion by tsuliwaensis** [Tampermonkey/tampermonkey#1598] [Q&A] [answered]
Is it possible to configure a Tampermonkey script so that it will apply only to the inner pages of a website? For example, I want the script to apply to `https://www.example.com/this-is-a-page?url=1` but not `https://www.example.com/`. In other words, the script will not apply to the root. I tried playing with _@ match_, _User Includes_, and _User Excludes_ without any success.

**Comment by derjanb** [comment, marked as answer]
This should do what you want...

```JS
// ==UserScript==
// @name         Exclude Root
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @match        https://www.example.com/*
// @exclude      https://www.example.com/
// @grant        none
// ==/UserScript==

console.log('running');
```

**Reply by tsuliwaensis** [reply]
Well, that was obvious in hindsight. Thank you so much!
